src/client/auth/auth.go

```diff
@@ -68,26 +69,26 @@ var (
 	//
 	// Note: This error message string is matched in the UI. If edited,
 	// it also needs to be updated in the UI code
-	ErrNotActivated = errors.New("the auth service is not activated")
+	ErrNotActivated = status.Error(codes.Unimplemented, "the auth service is not activated")
 
 	// ErrPartiallyActivated is returned by the auth API to indicated that it's
 	// in an intermediate state (in this state, users can retry Activate() or
 	// revert with Deactivate(), but not much else)
-	ErrPartiallyActivated = errors.New("the auth service is partially activated")
+	ErrPartiallyActivated = status.Error(codes.Unavailable, "the auth service is partially activated")
 
 	// ErrNotSignedIn indicates that the caller isn't signed in
 	//
 	// Note: This error message string is matched in the UI. If edited,
 	// it also needs to be updated in the UI code
-	ErrNotSignedIn = errors.New("auth token not found in context (user may not be signed in)")
+	ErrNotSignedIn = status.Error(codes.Unauthenticated, "no authentication token in context (try logging in)")
```

Comment:
Could we also take this opportunity to make these error messages more targeted at things users understand?
In particular I think the word `context` here is unlikely to mean anything to our non go-developer users. Something like "not authenticated, try logging in"  would be a bit better though. The one below is probably even a bit more confusing because it doesn't give them any suggested actions to take to remedy the problem.

Reply:
Done, let me know what you think

Reply:
:+1: